packages/hint-compat-api/src/css.ts

```diff
@@ -24,6 +24,7 @@ type ReportData = {
     formatFeature?: (name: string) => string;
     isValue?: boolean;
     node: ChildNode;
+    severity?: Severity;
```

Comment:
I don't think I've seen any `ReportData` with a custom `severity`. If we aren't using it maybe we should just delete this?

Reply:
Agreed. This is going to require a different approach anyway as the current implementation always ends up with a severity of `warning`. That didn't get caught in CI due to an issue in the test runner which I just posted a fix for in #3310.

@sarvaje I think the easiest way to flip between `warning` and `error` is to check in the `report` function if `formatAlternatives` returns a non-empty array. If so, that should be an `error`, otherwise a `warning`.